website/docs/docs/collaborate/git/version-control-basics.md

```diff
@@ -0,0 +1,70 @@
+| Commit | A commit is a user’s change to a file (or set of files). When you make a commit to save your work, Git creates a unique ID that allows you to keep a record of the specific changes committed along with who made them and when. Commits usually contain a commit message which is a brief description of what changes were made. |
+| Main | The primary, base branch of all repositories. All committed and accepted changes should be on the Main (or master) branch. In the Cloud IDE, the “Main” branch will be read-only. This is because any changes/edits to code cannot and should not be made directly in the base branch. A new branch should be created in order to make any changes to your project |
+| Merge | Merge takes the changes from one branch and adds them into another (usually main) branch. These commits are usually first requested via pull request before being merged by a maintainer. |
+| Pull Request | If someone has changed code on a separate branch of a project and wants it to be reviewed to add to the main branch, they can submit a pull request. Pull requests ask the repo maintainers to review the commits made, and then, if acceptable, merge the changes upstream. A pull happens when adding the changes to the main branch. |
+| Push | A push updates a remote branch with the commits made to the current branch. You are literally “pushing” your changes onto the remote. |
+| Remote | This is the version of a repository or branch that is hosted on a server. Remote versions can be connected to local clones so that changes can be synced. |
+
+
+## The git button in the Cloud IDE
+
+The git button in the Cloud IDE allows you to apply the concept of version control to your project. This page provides descriptions of each git button command and what they do:
+
+
+| Name | Actions |
+| --- | --- |
+| Abort merge | This option allows you to cancel a merge that had conflicts. Please note that all the changes will be reset, and this operation cannot be reverted, so make sure to commit or save all your changes before you start a merge. |
+| Change branch | This option will allow you to change between branches (checkout). |
+| Commit and push | Committing is similar to saving any changes made within your repo. In the above situation, the changes being saved or committed are the initialization of the project. The required files and folders are being added. When you make changes to your code in the future, you'll need to commit them as well. This allows you to record what changes were made when they were made, and who made them. |
+| Create new branch | This allows you to branch off of your base branch and edit your project. You’ll notice after initializing your project that the “main branch will be read-only. This is because any changes to code cannot and should not be made directly in the base branch. A new branch should be created in order to make any changes to your project. |
```

Comment:
```suggestion
| Create new branch | This allows you to branch off of your base branch and edit your project. You’ll notice after initializing your project that the main branch will be read-only. This is because any changes to code cannot and should not be made directly in the base branch. A new branch should be created in order to make any changes to your project. |
```